package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/sg.js",
   "scripts": {
     "prepublish": "BABEL_ENV=dist node_modules/babel-cli/bin/babel.js src --out-dir dist --ignore spec.js",
-    "test": "BABEL_ENV=dev node_modules/mocha/bin/mocha ./test/ ./src/**/*.spec.js"
+    "test": "BABEL_ENV=dev node_modules/mocha/bin/mocha ./test/ ./src/**/*.spec.js ./src/**/**/*.spec.js"
```

Comment:
If I remember correctly, you can put quotes around `./src/**/*.spec.js` and avoid the added part